src/main/java/frc/robot/RobotContainer.java

```diff
@@ -67,6 +86,33 @@ public RobotContainer() {
             },
             m_robotDrive));
 
+    new Trigger(() -> triggerPressed())
+        .whileTrue(new BasicDriveCommand(m_robotDrive, m_driverController));
+
+    // RunIntake constructor boolean is whether or not the intake should run reversed.
+    new Trigger(this::getIntakeButton).whileTrue(new RunIntake(m_intake, false));
+    new Trigger(this::getReverseIntakeButton).whileTrue(new RunIntake(m_intake, true));
+    // just shoot on trigger
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.kShoot))
+        .whileTrue(new Shoot(m_indexer, false));
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.kShootReverse))
+        .whileTrue(new Shoot(m_indexer, true));
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.kAimAmp))
+        .whileTrue(new Aim(m_shooter, FieldElement.AMP));
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.kAimSpeaker))
+        .whileTrue(new Aim(m_shooter, FieldElement.SPEAKER));
+
+    m_trapAim.whileTrue(new Aim(m_shooter, FieldElement.TRAP));
+
+    // triggers for extending and retracting shield manually
+    // don't extend shield
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.kExtendShield))
+        .onTrue(new ActuateShield(m_shooter, false));
+    // extend shield
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.kRetractShield))
+        .onTrue(new ActuateShield(m_shooter, true));
+
+    autoChooser.setDefaultOption("Leave Top", AutoBuilder.buildAuto("LeaveFromTop"));
```

Comment:
what does this auto do? is it tested.

Reply:
none are tested but this one just leaves